source/common/common/fine_grain_logger.cc

```diff
@@ -162,10 +161,22 @@ spdlog::logger* FineGrainLogContext::createLogger(const std::string& key)
   return new_logger.get();
 }
 
+void FineGrainLogContext::updateVerbosityDefaultLevel(level_enum level) {
+  absl::WriterMutexLock ul(&fine_grain_log_lock_);
+  if (level == verbosity_default_level_) {
+    return;
+  }
+
+  verbosity_default_level_ = level;
+  for (auto& [key, logger] : *fine_grain_log_map_) {
+    logger->set_level(getLogLevel(key));
```

Comment:
likewise, if verbosity_update_info is empty, we don't need to query any of these levels, and can simply set them. There's an existing `setAllFineGrainLoggers` that does this